### ISSUE TITLE
Attribute error running `pipenv install`

### ISSUE THREAD

**Issue by frankh** [kennethreitz/pipenv#170]
My Pipfile:

```python[[source]]
url = "https://pypi.python.org/simple"
verify_ssl = true

[dev-packages]
coverage = "*"
mock = "*"
django-coverage-plugin = "*"

[packages]
django-appconf = "*"
gevent = "*"
icalendar = "*"
django-auth-ldap = "*"
amqp = "*"
psycopg2 = "*"
django-jsonify = "*"
djangorestframework = "*"
django-redis = "*"
gunicorn = "*"
pytz = "*"
django-smtp-ssl = "*"
httplib2 = "*"
twilio = "*"
python-dateutil = "*"
argparse = "*"
wsgiref = "*"
django-filter = "*"
django-celery = "*"
requests = "*"
anyjson = "*"
pyasn1 = "*"
psycogreen = "*"
dj-database-url = "*"
ndg-httpsclient = "*"
pyjwt = "*"
django-compressor = "*"
django = "==1.6.11"
south = "*"
markdown = "*"
pyopenssl = "*"
django-polymorphic = "*"

[requires]
python_version = "2.7"

[packages.celery]
extras = [ "redis",]
```

Running `pipenv install` gives the error:

```

Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 669, in install
    do_init(dev=dev, requirements=requirements, allow_global=system)
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 500, in do_init
    do_lock()
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 390, in do_lock
    results = get_downloads_info(names_map, 'packages')
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 352, in get_downloads_info
    if is_required_version(version, specified_version):
  File "/usr/local/lib/python2.7/site-packages/pipenv/utils.py", line 138, in is_required_version
    if specified_version.startswith('=='):
AttributeError: 'OrderedDict' object has no attribute 'startswith'
```

**Comment by jungornti** [comment]
You have old version of `pipenv`, try to update. This was fixed.

**Comment by frankh** [comment]
I just updated it

```
$ pipenv --version
pipenv, version 3.2.11
```

**Comment by nateprewitt** [comment]
Thanks for opening a issue, @frankh. This was indeed fixed in 3.2.13 with 6888b58, please update your pipenv client. This is still a very young project, so we've been iterating fairly rapidly trying to strike a balance between features and stability. Occasionally, some things slip through.

I'm closing this for now, but please feel free to comment on the original issue (#156) and we can look at reopening it if the issue persists.
